simulator/src/replay/cmd_execute_in_memory.rs

```diff
@@ -38,6 +42,18 @@ impl TxnExecuteInMemory {
             Some(n) => NetworkDefinition::from_str(n).map_err(Error::ParseNetworkError)?,
             None => NetworkDefinition::mainnet(),
         };
+        let mut breakpoints = BTreeMap::<u64, Hash>::new();
+        if let Some(bps) = &self.breakpoints {
+            for bp in bps.split(",") {
+                let mut tokens = bp.trim().split(":");
+                if let Some(version) = tokens.next().and_then(|x| u64::from_str(x).ok()) {
+                    if let Some(hash) = tokens.next().and_then(|x| Hash::from_str(x).ok()) {
```

Comment:
Perhaps better to error out if decoding input fails?

Reply:
Yep